lib/PostMessageTransport.js

```diff
@@ -90,4 +109,5 @@ define(function (require, exports, module) {
     module.exports.start           = start;
     module.exports.send            = send;
     module.exports.close           = close;
+    module.exports.reload          = reload;
```

Comment:
It's called in [here] (https://github.com/K-LV/brackets-browser-livedev/blob/patchReload/main.js#L149)